### PR TITLE
docs: ccd prometheus added

### PR DESCRIPTION
**Pull request by bvdeenen**

### CODE REVIEW

File: docs/quickstart/ccd.md [hunk 1]
```diff
@@ -310,3 +310,41 @@ If you've made mistakes and want to start over:
...
+
+## Monitoring the cluster via Prometheus
+Most components of STRM Privacy expose [Prometheus][prometheus] metrics and we've included the [Prometheus Cluster
+Operator][prom-helm] Helm chart. In order to define _scrape targets_ to Prometheus, we deploy  [Service
+Monitors][prom-sm] that expose the metrics endpoints in the Event Gateway.
+
+### the Prometheus server
+Access the Prometheus server either via [telepresence][telepresence] or via a port-forward
+
+    kubectl port-forward service/strmprivacy-kube-prometheu-prometheus 9090:9090&
```
Comment (ivan-p92) [inline docs/quickstart/ccd.md:325]: Typo in the service name.

File: docs/quickstart/ccd.md [hunk 2]
```diff
@@ -310,3 +310,41 @@ If you've made mistakes and want to start over:
...
+
+
+### Grafana
+The CCD quickstart also includes a Grafana instance, that we can port-forward
+
+    kubectl port-forward service/strmprivacy-grafana 4000:80&
+
+I've added two panels for the very simple dashboard we show here:
+
+![grafana](images/ccd/grafana.png)
```
Comment (ivan-p92) [inline docs/quickstart/ccd.md:348]: Perhaps we can make the y axis and the legend of these graphs more readable?

**Review by ivan-p92 — APPROVED** [review]

File: docs/quickstart/ccd.md (hunk 1 above)
Comment (bvdeenen) [reply docs/quickstart/ccd.md:325]: That is actually done like this by the prometheus operator, to keep the service name short enough :-)

File: docs/quickstart/ccd.md (hunk 2 above)
Comment (bvdeenen) [reply docs/quickstart/ccd.md:348]: I agree, but want to pick this up in a separate monitoring chapter of the documentation, and I think we'll get to Grafana-as-a-service (via the SAAS control plane)

File: docs/quickstart/ccd.md (hunk 1 above)
Comment (ivan-p92) [reply docs/quickstart/ccd.md:325]: Fascinating 😅

**Pull request merged**